samples/Nancy.Demo.Async/project.json

```diff
@@ -0,0 +1,25 @@
+﻿{
+  "version": "2.0.0-*",
+
+  "description": "Nancy is a lightweight web framework for the .Net platform, inspired by Sinatra. Nancy aim at delivering a low ceremony approach to building light, fast web applications.",

```

Comment:
I'm submitting the rest without them. I'm not bothering coming up with sensible descriptions and tags for all the demo projects.